src/components/Login.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component, Fragment} from 'react';
+import {View, Text, Button} from 'react-native';
+import {Input, Loading} from './common';
+import deviceStorage from '../services/deviceStorage';
+import axios from 'axios';
+
+export default class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            email: '',
+            password: '',
+            error: '',
+            loading: false
+        };
+        this.validateJson = this.validateJson.bind(this);
+        this.loginUser = this.loginUser.bind(this);
+    }
+
+    validateJson(json) {
+        if (json.data.hasOwnProperty('jwt')) {
+            deviceStorage.saveJWT("id_token", json.data.jwt);
+            this.props.newJWT(json.data.jwt);
+        }
+        else {
+            this.setState({error: json.data.error, loading: false});
+            console.log(json);
+        }
+    }
+
+    loginUser() {
+        const API_CALL = 'https://sei19-safira.herokuapp.com/api/auth/sign_in';
```

Comment:
Move this to a `.env` file.

Reply:
Yes, i dont know if a .env is the standard in react native but a equivalent option is the best choice for easy deployment